libpod/util.go

```diff
@@ -13,6 +15,8 @@ import (
 	"strings"
 	"time"
 
+	units "github.com/docker/go-units"
+
```

Comment:
please include it as part of the other imports block

Reply:
fixed.